src/components/v2/appDetails/sourceInfo/environmentStatus/NotesDrawer.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef } from 'react'
+import { Drawer } from '../../../../common'
+import { ReactComponent as Close } from '../../../assets/icons/ic-close.svg'
+import { MarkDown } from '../../../../charts/discoverChartDetail/DiscoverChartDetails'
+import './environmentStatus.scss'
+
+function NotesDrawer({ notes, close }: { notes: string; close: () => void }) {
+    const appNotesRef = useRef<HTMLDivElement>(null)
+
+    const escKeyPressHandler = (evt): void => {
+        if (evt && evt.key === 'Escape' && typeof close === 'function') {
+            evt.preventDefault()
+            close()
+        }
+    }
+
+    const outsideClickHandler = (evt): void => {
+        if (
+            appNotesRef.current &&
+            !appNotesRef.current.contains(evt.target) &&
+            typeof close === 'function'
```

Comment:
We can avoid this check, It seems unnecessary

Reply:
Yeah, you're right :)
Removed the `typeof` check.